Makefile

```diff
@@ -1,6 +1,6 @@
 FIG=docker-compose
-RUN=$(FIG) run --rm app
-EXEC=$(FIG) exec app
+RUN?=$(FIG) run --rm app
+EXEC?=$(FIG) exec app
```

Comment:
Can you fix this in another PR ?

Reply:
Sure -> see https://github.com/EnMarche/en-marche.fr/pull/1833